src/components/Nav/BackgroundDropdownItem.tsx

```diff
@@ -3,17 +3,20 @@ export const BackgroundDropdownItem = ({
   setIsBackground,
   background,
   title,
+  className,
 }: {
   isPicked: boolean;
   setIsBackground: any;
   background: any;
   title: string;
+  className?: string;
 }) => {
   return (
     <div
       className={
-        "cursor-pointer py-1 hover:rounded-md hover:bg-gray-100 dark:hover:bg-gray-900 " +
-        (isPicked && " rounded-md bg-gray-100 dark:bg-gray-900")
+        className +
```

Comment:
I'm down with adding the `className` prop, but with that, the regular classes and the `isPicked` that component is getting messy. I think adding the `clsx` package to the repo, then doing something like the following is far easier to read:

```
className={clsx(
    className,
    "cursor-pointer py-1 bg-gray-300 dark:bg-gray-800 hover:bg-gray-100 dark:hover:bg-gray-900",
    isPicked && "bg-gray-100 dark:bg-gray-900"
)}
```

@Melkeydev Thoughts? Also we can merge this PR and I can go through and add clsx and update code later. Probably better to do it that way in fact than worrying about it in this PR.

Reply:
I dont mind className prop for right now. 
I made an issue #214 for future